### PR TITLE
Feedback rest api 2320173

### PR DESCRIPTION
**Pull request by ldonjibson**
waiting for review

### CODE REVIEW

File: server/controllers/authentication.js [hunk 1]
```diff
@@ -67,18 +66,27 @@ router.post('/auth/signup', (req, res) => {
...
 		}
-		// console.log(req.body)
 
 		//This pushes the newly gotten value to the data
-		// users = JSON.stringify(usobj);
 		let usObj = users
 		usObj.push(newUser);
-		// console.log(usObj);
 		const getUser = usObj.find(usr => usr.id === Number(usObj.length));
 		delete getUser['password'];
```
Comment (slimsolz) [inline server/controllers/authentication.js]: is there a reason to delete the user's password??

File: server/controllers/authentication.js [hunk 2]
```diff
@@ -108,15 +116,14 @@ router.post('/auth/signin', (req, res) => {
 		const getUser = users.find(usr => usr.email === email);
 		if (!getUser){
 			res.json({
-				"status": 1002,
+				"status": 400,
```
Comment (slimsolz) [inline server/controllers/authentication.js]: I think the correct status code for "not found" is 404

File: server/index.js [hunk 3]
```diff
@@ -54,7 +27,6 @@ server.use(morgan('combined'));
 
 
 server.get('/api/v1/', (req, res) =>{
-	// console.log(res)
 	let content = {
 		status: 1000,
```
Comment (slimsolz) [inline server/index.js]: change this code

File: server/middleware/verifyStaff.js [hunk 4]
```diff
@@ -37,7 +37,7 @@ const jwtStaffVerify= ((req, res, next) => {
 					next();
 				} else {
 					res.json({
-						"status":1005,
+						"status":401,
```
Comment (slimsolz) [inline server/middleware/verifyStaff.js]: use 403 here

File: server/routes/adminRoute.js [hunk 5]
```diff
@@ -98,27 +99,27 @@ router.patch('/allusers/profile/:id/edit', upload.upload.single('file'), jwtAdmi
 		}
 		if (!image){
 			res.json({
-				"status": 1000,
+				"status": 201,
```
Comment (slimsolz) [inline server/routes/adminRoute.js]: for update use 200

File: server/routes/adminRoute.js [hunk 6]
```diff
@@ -129,25 +130,25 @@ router.patch('/allusers/profile/:id/changepassword',jwtAdminVerify,  (req, res)
 				let hashedPassword = bcrypt.hashSync(password, 8);
 				chKUser.password = hashedPassword
 				res.json({
-					"status": 1000,
+					"status": 201,
```
Comment (slimsolz) [inline server/routes/adminRoute.js]: password change use 200

File: server/routes/adminRoute.js [hunk 7]
```diff
@@ -129,25 +130,25 @@ router.patch('/allusers/profile/:id/changepassword',jwtAdminVerify,  (req, res)
...
 					"message": "Password changed successfully"
 				});
 			} else{
 				res.json({
-					"status": 1008,
+					"status": 401,
```
Comment (slimsolz) [inline server/routes/adminRoute.js]: 400

File: server/controllers/authentication.js (hunk 1 above)
Comment (ldonjibson) [reply server/controllers/authentication.js]: i will fix the delete thing

File: server/routes/adminRoute.js (hunk 5 above)
Comment (ldonjibson) [reply server/routes/adminRoute.js]: okay

File: server/routes/adminRoute.js (hunk 6 above)
Comment (ldonjibson) [reply server/routes/adminRoute.js]: oaky

File: server/routes/adminRoute.js (hunk 7 above)
Comment (ldonjibson) [reply server/routes/adminRoute.js]: oaky

**Review by slimsolz — APPROVED** [review]

**Pull request merged**